aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -45,22 +45,34 @@ static inline std::tuple<Tensor, Tensor> _lu_det_P_diag_U(const Tensor& self) {
   return std::tuple<Tensor, Tensor>(num_exchanges.mul_(-2).add_(1), u_diagonal);
 }
 
-// torch.linalg.det, alias for torch.det
-Tensor linalg_det(const Tensor& self) {
-  return self.det();
+// torch.det, alias for torch.linalg.det
+Tensor det(const Tensor& self) {
+  return at::linalg_det(self);
 }
 
-Tensor det(const Tensor& self) {
+Tensor& linalg_det_out(const Tensor& self, Tensor& out) {
+  checkSameDevice("linalg_det", out, self, "out");
```

Comment:
Let's make sure to use nice user facing names in error messages like "torch.linalg.det()" instead of "linalg_det"

Reply:
good idea, fixed by d29e104a9f